contrib/online_ide/sources/nit/pnacl_nit.nit

```diff
@@ -82,7 +82,8 @@ redef class FileReader
 
 	redef fun fill_buffer
 	do
-		_buffer.clear
+		_buffer_pos = 0
+		_buffer_length = 0
```

Comment:
Since you seems to use this at some other place, I suggest that you provide a private `buffer_clear` method to factorize and improve the code